consensus/src/consensus.rs

```diff
@@ -129,14 +188,17 @@ where
     pub fn spawn(
         committee: SharedCommittee<PublicKey>,
         store: Arc<ConsensusStore<PublicKey>>,
+        cert_store: Store<CertificateDigest, Certificate<PublicKey>>,
         rx_primary: Receiver<Certificate<PublicKey>>,
         tx_primary: Sender<ConsensusPrimaryMessage<PublicKey>>,
         tx_output: Sender<ConsensusOutput<PublicKey>>,
         protocol: Protocol,
         metrics: Arc<ConsensusMetrics>,
+        gc_depth: Round,
     ) -> JoinHandle<StoreResult<()>> {
         tokio::spawn(async move {
             let consensus_index = store.read_last_consensus_index()?;
+            let recovered_last_committed = store.read_last_committed();
```

Comment:
Nit: while the name `recovered_last_committed` makes sense to me, `recover_last_committed` less so.